e2e/test-specs.yml

```diff
@@ -0,0 +1,92 @@
+description: |
+  End-to-end tests for k8s integration
+
+custom_test_key: k8s.clusterName
+
+scenarios:
+  - description: |
+      This scenario will verify that metrics from a k8s Cluster are correctly collected.
+    before:
+      - helm upgrade --install ${SCENARIO_TAG}-resources -n nr-${SCENARIO_TAG} --create-namespace ./charts/e2e-resources
+      - helm upgrade --install ${SCENARIO_TAG} -n nr-${SCENARIO_TAG} --create-namespace newrelic/nri-bundle --values e2e-values.yml --set global.licenseKey=${LICENSE_KEY} --set global.cluster=${SCENARIO_TAG}
+    after:
+      - kubectl logs -l app=newrelic-infrastructure
+      - helm delete ${SCENARIO_TAG}-resources
+      - helm delete ${SCENARIO_TAG}
+    tests:
+      nrqls: []
+      entities: []
+      metrics:
+        - source: "k8s.yml"
+          except_entities: []
+          except_metrics:
+            - k8s.container.cpuCoresUtilization
+            - k8s.container.cpuLimitCores
```

Comment:
if we add the resource limits to the infra pod  these two metrics should appear right?

Reply:
definetely! I wanted to track down why those were not available and you found it!